src/Term/TermList.php

```diff
@@ -190,4 +190,13 @@ public function hasTerm( Term $term ) {
 			&& $this->terms[$term->getLanguageCode()]->equals( $term );
 	}
 
+	/**
+	 * Removes all terms from this list.
+	 *
+	 * @since 5.1
+	 */
+	public function clear() {
```

Comment:
Yes, when we have entities that don't contain a `Fingerprint` but only a `TermList` or an `AliasGroupList`. We definitely need the `clear` method there.